### PR TITLE
add npm badges for all modules

### LINKED ISSUE #264: Add one-line simple explaination for devtools-modules and devtools-sham-modules
Currently there's no description for what these modules responsible for and what are the differences between these two modules.

### PR DESCRIPTION
**Pull request by gasolin**
Link all modules current npm badge at readme to easy checking the dependencies

demo https://github.com/gasolin/devtools-core/tree/badges

also fix #264

### CODE REVIEW

**Comment by jasonLaster** [comment]
👍

File: packages/devtools-modules/index.js [hunk 1]
```diff
@@ -2,12 +2,20 @@ const Services = require("./client/shared/shim/Services");
 const SplitBox = require("./client/shared/components/splitter/SplitBox");
 // const SplitBoxCSS = require("./client/shared/components/splitter/SplitBox.css")
 const sprintf = require("./shared/sprintf").sprintf;
+const Tabbar = require("./client/shared/components/tabs/tabbar");
+const TabPanel = require("./client/shared/components/tabs/tabs").TabPanel;
+const TreeRow = require("./client/shared/components/tree/tree-row");
+const TreeView = require("./client/shared/components/tree/tree-view");
```
Comment (jryans) [inline packages/devtools-modules/index.js]: How did these changes end up here? They seem unrelated to NPM badges, and they also reference files that don't exist (the tabs ones).

I'll revert the changes to this file for now.

Comment (jryans) [inline packages/devtools-modules/index.js]: Reverted in 4482a0b. Sorry for the revert, but this was breaking build steps in other packages.

Comment (jasonLaster) [inline packages/devtools-modules/index.js]: Thanks for reverting @jryans 

It looks like that patch accidentally crept in there. The dependencies are in `devtools-sham-modules`

https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-sham-modules/index.js

Comment (gasolin) [inline packages/devtools-modules/index.js]: @jryans Thanks for catching that, I'll make another patch with clear dependency

**Pull request merged**